modules/javafx.base/src/test/java/test/com/sun/javafx/binding/ErrorLoggingUtility.java

```diff
@@ -25,15 +25,15 @@
 
 package test.com.sun.javafx.binding;
 
-import static org.junit.jupiter.api.Assertions.*;
-
-import com.sun.javafx.logging.PlatformLogger.Level;
-
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import com.sun.javafx.binding.Logging;
-import com.sun.javafx.binding.Logging.ErrorLogger.ErrorLogRecord;
 import com.sun.javafx.binding.Logging.ErrorLogger;
+import com.sun.javafx.binding.Logging.ErrorLogger.ErrorLogRecord;
+import com.sun.javafx.logging.PlatformLogger.Level;
 
-public class ErrorLoggingUtiltity {
+public class ErrorLoggingUtility {
```

Comment:
I see you left in the name change to `ErrorLoggingUtility`. I was hoping you'd revert that along with the other changes to this file, since they are now unrelated and make the PR much larger to review. Yes, I know it's a mechanical change, but I find it hard to see what has "really" changed now.

Suggestion: revert the name change to this utility, since it is no longer relevant.